client/packages/requisitions/src/ResponseRequisition/api/hooks/line/useResponseLines.ts

```diff
@@ -1,45 +1,8 @@
-import { useMemo } from 'react';
-import {
-  SortController,
-  SortUtils,
-  Column,
-  useItemUtils,
-} from '@openmsupply-client/common';
 import { ResponseLineFragment } from '../../operations.generated';
-import { useResponseColumns } from '../../../DetailView/columns';
 import { useResponseFields } from '../document/useResponseFields';
 
-interface UseResponseLinesController
-  extends SortController<ResponseLineFragment> {
-  lines: ResponseLineFragment[];
-  columns: Column<ResponseLineFragment>[];
-  itemFilter: string;
-  setItemFilter: (itemFilter: string) => void;
-  onChangeSortBy: any;
-}
-
-export const useResponseLines = (): UseResponseLinesController => {
+export const useResponseLines = (): ResponseLineFragment[] => {
```

Comment:
This whole hook is pretty much redundant now. But best to leave it until we re-factor the API hooks for this area -- the whole thing will get a lot simpler.